.gitignore

```diff
@@ -1,27 +1,58 @@
-# General
+target/
```

Comment:
There were already .gitignores in the subdirectories but if. you want to have a one more I'm okay :)

Reply:
I expected this comment :)  I agree here. I did fill it out automatically at the very start once I saw that .gitignore file was empty and later realized it has no relations really. Same time thought not a big minus to keep it anyway in any case.  I would leave it unless you mean to not do it.

Reply:
it does not harm lets leave it then.